introduction.md

```diff
@@ -1,2 +1,12 @@
 Hi, my name is Nicolas and i'm learning github!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?